### PR TITLE
Post section

### PR DESCRIPTION
**Pull request by Ravyar19**

### CODE REVIEW

**Review by murtaja1 — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 
+
 import React from 'react'
 export default function App() {
 	return (
 		<div>			
 			
+
+
```
Comment (murtaja1) [inline src/App.js:9]: this div should be a closing tag.

**Review by murtaja1 — CHANGES_REQUESTED** [review]

File: src/components/PostsSection/PostsSection.jsx [hunk 2]
```diff
@@ -91,22 +54,24 @@ export default function Posts() {
 
                 <Container className={classes.cardGrid} maxWidth="lg">
                     <Grid container spacing={3}>
-                        {item.slice(0, 4).map((data)=>(
+                        {posts.map((data)=>(
```
Comment (murtaja1) [inline src/components/PostsSection/PostsSection.jsx]: add key to the grid.

**Review by murtaja1 — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
 import React from 'react'
-import Container from './containers/Container'
 
 export default function App() {
 	return (
+		<div>			
+			
+
+
```
Comment (murtaja1) [inline src/App.js:9]: add closing tag for div.

**Review by murtaja1 — CHANGES_REQUESTED** [review]

**Review by ZhinAbubakr — APPROVED** [review]

**Review by maqalaqil — APPROVED** [review]
LGTM

**Review by murtaja1 — APPROVED** [review]

**Pull request merged**